vendor/github.com/kubernetes-csi/drivers/pkg/csi-common/identityserver-default.go

```diff
@@ -56,7 +56,7 @@ func (ids *DefaultIdentityServer) GetPluginCapabilities(ctx context.Context, req
 			{
 				Type: &csi.PluginCapability_Service_{
 					Service: &csi.PluginCapability_Service{
-						Type: csi.PluginCapability_Service_UNKNOWN,
```

Comment:
you shouldn't change vendor code directly.

Reply:
sure, I'll revert that